lecsum-fe/pages/retry_exam.py

```
@@ -0,0 +1,474 @@
+        else:
+            st.error(f"채점 실패: {response.status_code} - {response.text}")
+            return None
+    except Exception as e:
+        st.error(f"서버 연결 실패: {e}")
+        return None
+
+
+def fetch_retry_attempts(limit=50):
+    """재시험 응시 기록 조회"""
+    try:
+        response = requests.get(
```

Ruff (0.14.8)

Probable use of `requests` call without timeout

(S113)